include/ultra64/gbi.f3dex3.h

```diff
@@ -1204,7 +1204,8 @@ typedef struct {
 } OcclusionPlaneCandidate;
 
 typedef union {
-    Light_t l;
+    Light_t    l;
+    PosLight_t p;
```

Comment:
I'm happy to send this upstream too. I prefer to use the existing `Light` type so that it's not different between F3DEX2 (with point lighting extensions) and F3DEX3.

Reply:
But the F3DEX3 GBI is already roughly the same as the F3DEX2 point lights GBI... What you've created is different from them.

Reply:
Moving to the other thread below.